### PR TITLE
Removed qspector, prepared for bioconda build pipes, package name to lowercase

### PR DESCRIPTION
**Pull request by ypriverol**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
-    
+
 setup(
-    name='QCCalculator',
+    name='qccalculator',
     version='0.0.2',
     packages=find_packages(),
     url='https://gitlab.ebi.ac.uk/walzer/qccalculator',
-    description='Development for python driven QC calculation (QCCalculator) and presentation (QSpector)',
+    description='Development for python driven QC calculation (qccalculator) and presentation (QSpector)',
```
Comment (mwalzer) [inline setup.py]: Can you remove the QSpector bit?

File: setup.py [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
     long_description=long_description,
     install_requires=[
```
Comment (mwalzer) [inline setup.py:13]: Is there a way to import the requirements.txt?

Comment (mwalzer) [reply setup.py:13]: Then we would only have to maintain one file in case of dependency changes

File: setup.py [hunk 3]
```diff
@@ -22,8 +22,7 @@
         "toposort"
         ],
     include_package_data=True,
-    #scripts=['bin/qspector/qspector.py'] ,
     entry_points={
-        'console_scripts': ['QCCalculator = QCCalculator.cli:start']
+        'console_scripts': ['qccalculator = qccalculator.cli:start']
```
Comment (mwalzer) [inline setup.py]: This also changes the executables name to lower case. I'd prefer the exec to be called QCCalculator to distinguish from the python package. Same as in `ThermoRawFileParser.exe`:
`'console_scripts': ['QCCalculator = qccalculator.cli:start']`

File: README.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # QCCalculator
 
-Development for python driven QC calculation (QCCalculator) and presentation (QSpector)
+Development for python driven QC calculation (QCCalculator)
 
-## Install 
+## Install
 pip install -U git+https://github.com/bigbio/qccalculator.git#egg=QCCalculator
```
Comment (mwalzer) [inline README.md:6]: I think the egg name needs to be same as package name -> lower case

File: qccalculator/cli.py [hunk 5]
```diff
@@ -27,57 +28,57 @@ def finale():
         else:
             with open(out, 'w') as fh:
                 fh.write(qc.JsonSerialisable.ToJson(mzqc_assembly(rqs, sqs, out), readability=0))
-    logging.warn("Done. Thank you for choosing QCCalculator!")
+    logging.warn("Done. Thank you for choosing qccalculator!")
```
Comment (mwalzer) [inline qccalculator/cli.py]: Tool name (camel case)

File: qccalculator/cli.py [hunk 6]
```diff
@@ -27,57 +28,57 @@ def finale():
...
 
 @click.group(chain=True)
 @click.option('--output', required=True, type=click.Path(), default="/tmp/out.mzQC", help="The path and name of the desired output file.")
 @click.option('--zip/--no-zip', default=False, help="Apply gzip to the output. Appends '.gz' to the target filename and pretty formatting.")
 def start(output, zip):
-    """Calculate quality metrics for given files. 
+    """Calculate quality metrics for given files.
        Multiple files input is possible (each after a "full/basic" COMMAND).
-       All metrics of one QCCalculator execution will be stored in on output file. 
+       All metrics of one qccalculator execution will be stored in on output file.
```
Comment (mwalzer) [inline qccalculator/cli.py]: tool name

File: qccalculator/cli.py [hunk 7]
```diff
@@ -27,57 +28,57 @@ def finale():
...
        If you need separate mzQC files, please execute separately.
-       For more information on the different COMMAND types, try QCCalculator COMMAND --help"""
+       For more information on the different COMMAND types, try qccalculator COMMAND --help"""
```
Comment (mwalzer) [inline qccalculator/cli.py]: tool name

File: qccalculator/cli.py [hunk 8]
```diff
@@ -142,18 +144,18 @@ def maxq(filename, zipurl, rawname):
...
+    rq = basicqc.getBasicQuality(exp)
     rqs.append(rq)
 
     finale()
 
 if __name__ == "__main__":
     start()
-#QCCalculator --output cli-test.mzqc full --mzid tests/CPTAC_CompRef_00_iTRAQ_01_2Feb12_Cougar_11-10-09.mzid tests/CPTAC_CompRef_00_iTRAQ_01_2Feb12_Cougar_11-10-09.trfr.t3.mzML
\ No newline at end of file
+#qccalculator --output cli-test.mzqc full --mzid tests/CPTAC_CompRef_00_iTRAQ_01_2Feb12_Cougar_11-10-09.mzid tests/CPTAC_CompRef_00_iTRAQ_01_2Feb12_Cougar_11-10-09.trfr.t3.mzML
```
Comment (mwalzer) [inline qccalculator/cli.py]: tool name

**Review by mwalzer — CHANGES_REQUESTED** [review]
Can you make the description a bit more specific? Instead of 'major refactoring' something like: Removed qspector, prepared for bioconda build pipes, package name to lowercase

File: setup.py (hunk 2 above)
Comment (ypriverol) [reply setup.py:13]: @mwalzer I have been reading about this and is recommended to use both files. We can improve that in the future.

File: setup.py (hunk 3 above)
Comment (ypriverol) [reply setup.py]: Done

File: README.md (hunk 4 above)
Comment (ypriverol) [reply README.md:6]: Done

File: qccalculator/cli.py (hunk 8 above)
Comment (ypriverol) [reply qccalculator/cli.py]: Done

File: setup.py (hunk 2 above)
Comment (mwalzer) [reply setup.py:13]: okay

**Pull request merged**